wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSession+UserSession.swift

```diff
@@ -353,6 +354,23 @@ extension ZMUserSession: UserSession {
             lastE2EIUpdateDateRepository: lastE2EIUpdateDateRepository
         )
     }
+
+    public func makeAppendTextMessageUseCase() -> any AppendTextMessageUseCaseProtocol {
+        return AppendTextMessageUseCase(analyticsSession: analyticsSession)
+    }
+
+    public func makeAppendImageMessageUseCase() -> any AppendImageMessageUseCaseProtocol {
+        return AppendImageMessageUseCase(analyticsSession: analyticsSession)
+    }
+
+    public func makeAppendKnockMessageUseCase() -> any AppendKnockMessageUseCaseProtocol {
+        return AppendKnockMessageUseCase(analyticsSession: analyticsSession)
+    }
+
+    public func makeAppendLocationMessageUseCase() -> AppendLocationMessagekUseCaseProtocol {
```

Comment:
Why is this not `any`?

Reply:
I don't we need any in this case. In the first iteration we were using an associated type now we don't. Fixed it here: https://github.com/wireapp/wire-ios/commit/f16a68f8b2a651cda63a05d18394980f82791160

Reply:
I think they should all be marked with `any` to be explicit that these are existential types.

Reply:
Done: https://github.com/wireapp/wire-ios/commit/001b25009ce42dc70bd7307d4c7486a1ad078ff7